package.json

```diff
@@ -63,16 +64,16 @@
     "eslint-plugin-jsx-a11y": "^6.0.2",
     "eslint-plugin-react": "^7.5.1",
     "flow-bin": "^0.50.0",
-    "react": "^15.4.2",
+    "react": "^16.7.0",
     "react-addons-test-utils": "^15.6.2",
     "react-click-outside": "^3.0.0",
     "react-color": "^2.13.8",
-    "react-dom": "^15.4.2",
+    "react-dom": "^16.7.0",
     "webpack": "^3.8.1"
   },
   "peerDependencies": {
-    "react": "^0.14.0 || ^15.0.0 || ^16.0.0",
-    "react-dom": "^0.14.0 || ^15.0.0 || ^16.0.0"
```

Comment:
are there features that require react 16 and above? Would this not work with react 14 or 15?

Reply:
I was getting the [following issue](https://github.com/storybooks/storybook/issues/4796), and as per the suggestions there I upgraded React, and the issue resolved.